src/quo2/components/share/qr_code/view.cljs

```diff
@@ -1,11 +1,12 @@
 (ns quo2.components.share.qr-code.view
   (:require [quo2.components.share.qr-code.style :as style]
-            [react-native.core :as rn]))
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]))
 
 (defn qr-code
   [{:keys [source width height]}]
   [rn/view
    {:style style/container}
-   [rn/image
+   [fast-image/fast-image
```

Comment:
could you pls elaborate?

Reply:
`fast-image` gets rid of the flickering introduced when we change tabs. According to the author of the library they use "Aggressive Caching", ref : https://github.com/DylanVann/react-native-fast-image/blob/main/docs/how-is-caching-handled.md